.github/scripts/toolchain/build-gcc.sh

```diff
@@ -38,10 +38,10 @@ if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$GCC_BUILD_PATH/Makefile" ]]; then
                 # REMOVED: --libexecdir=/usr/lib
                 # CHANGED: --enable-__cxa_atexit to --disable-__cxa_atexit
                 TARGET_OPTIONS="$TARGET_OPTIONS \
+                    --enable-shared-libgcc \
```

Comment:
This was added to make the build configuration as close as possible to the upstream build.